assets/components/textInput/textInput.jsx

```diff
@@ -11,12 +11,14 @@ type PropTypes = {
   placeholder?: string,
   labelText?: string,
   id?: string,
+  onChange?: (name: string) => void,
+  value?: string,
 };
 
 
 // ----- Functions ----- //
 
-function inputClass(hasLabel) {
+function inputClass(hasLabel: boolean): string {
```

Comment:
Is it possible to add an Eslint rule to force that all the functions inside a file annotated with @flow must have type annotations?